Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ic-wasm"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
We are just adding new flags, no need for a major version bump?

Reply:
The old command no longer works so it's a breaking change.

`ic-wasm shrink --optimize` vs `ic-wasm optimize`